dropwizard-core/src/main/java/io/dropwizard/setup/Environment.java

```diff
@@ -51,7 +51,8 @@
     private final AdminEnvironment adminEnvironment;
```

Comment:
The file has increased in 1 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._